### ISSUE TITLE
Need a little bit of help with usage (err: Decryption process closed)

### ISSUE THREAD

**Issue by spinfal** [Geczy/ipa-bot#1]
Hey, I was hesitant to make an issue for this but I've been trying to troubleshoot and get things working but can't seem to figure this one issue out.

I have the Telegram bot all setup, yarn & node are installed, and so is everything else (Frida, AppSync, and OpenSSH is available with root). My iPhone X is on 16.1.2 and is jailbroken with palera1n.

When I run the command `/request <app link>` I run into this issue: 
![Screenshot of error in Terminal](https://cdn.spin.rip/r/WindowsTerminal_2142005474.png)

For my first time setting everything up, I used Windows 11. However for my second time I tried using Ubuntu on WSL-2 with no luck either (same issue on both Windows and WSL).

If you need any more information, feel free to ask and I'll provide.

**Comment by Geczy** [comment]
Try running the bash command manually, what do you see?

Next you'll have to try logging the output of bagbak in the bash script. I redirect that output to a variable to hide it right now

**Comment by Geczy** [comment]
Sounds like a line ending issue. Might have to add a gitattributes file https://docs.github.com/en/get-started/getting-started-with-git/configuring-git-to-handle-line-endings#refreshing-a-repository-after-changing-line-endings

You can try this locally

Did you connect your device over USB? It connects on a USB port via ssh I believe

**Comment by spinfal** [comment]
Thanks again for the response, I've figured out that the issue has to do with the fact that I'm using WSL. I've booted into a proper Ubuntu 22.04 installation and the bot works as expected.

**However**, the bot comes with Linux ARM support and MacOS support but no AMD support. This causes issues with Ubuntu (at least in my case), and I had to manually install amd64 from ipatool's repo and modify a sh file to use AMD (simply replaced arm64 with amd64) :slightly_smiling_face:
